docs/data/pages.ts

```diff
@@ -376,6 +393,17 @@ const pages: MuiPage[] = [
           { pathname: '/x/react-date-pickers/playground', title: 'Customization playground' },
         ],
       },
+      {
+        pathname: '/x/api/picker-resources',
+        subheader: 'Resources',
+        children: [
+          {
+            pathname: '/x/api/date-pickers-group',
+            title: 'API reference',
+            children: [{ pathname: '/x/api/date-pickers', title: 'Index' }, ...pickersComponentApi],
+          },
+        ],
+      },
```

Comment:
This introduced two bugs. Fixing this in https://github.com/mui/material-ui/pull/44514.